client/src/components/App.jsx

```diff
@@ -9,7 +9,7 @@ import '../style.css';
 const App = () => {
   const params = useParams();
   const id = params.id;
-  const [yourOutfit, setYourOutfit] = useState([]);
+  const [yourOutfit, setYourOutfit] = useState(JSON.parse(localStorage.getItem('yourOutfit')));
```

Comment:
Now the yourOutfit state will will be automatically set from localStorage (data saved in the user's browser) and adding / removing from yourOutfit will also update localStorage.

Reply:
This allows the user's outfit to be saved after existing and re-opening the website. You can look at localStorage by going to dev tools -> Application -> Local Storage as seen here:
[https://i.imgur.com/wDXr30z.png](https://i.imgur.com/wDXr30z.png)

Reply:
This looks great! I'm assuming I should change mine to also just put the product in localStorage as well?

Reply:
You won't need to, as the addToOutfit function in App.jsx will handle that!